addons/pos_loyalty/static/src/js/ProductScreen.js

```diff
@@ -225,6 +225,12 @@ export const PosLoyaltyProductScreen = (ProductScreen) =>
                 selectedLine.order._updateRewards();
             }
         }
+        async _showDecreaseQuantityPopup() {
+            const result = await super._showDecreaseQuantityPopup();
+            if (result){
+                this.env.pos.get_order()._updateRewards();
```

Comment:
```suggestions
                this.env.pos.get_order()?._updateRewards()
```
is the `?.` access already being used in 16.0? If so, use it. Otherwise, you need to check if the get_order returns a value.

Reply:
Is it possible to call `_showDecreaseQuantityPopup` when there isn't any order?

Reply:
You're right. In that context, order should always be present. In fact, we should allow it to crash if it's not present. Meaning, something is wrong above the call stack.